hack/app_sre_pr_check.sh

```diff
@@ -4,6 +4,20 @@
 
 set -exv
 
+CURRENT_DIR=$(dirname "$0")
+CRD_DIR="$CURRENT_DIR"/../deploy/crds
+
```

Comment:
We should add support for also looking in the manifests folder

Reply:
Done